### DISCUSSION TITLE
Placement attribute gone from IStepOption?

### DISCUSSION THREAD

**Discussion by JoramM** [hakimio/ngx-ui-tour#136] [Q&A] [answered]
Hey,
I saw that in this [commit](https://github.com/hakimio/ngx-ui-tour/commit/9b18bad731ed6cfe49b321c11743f470b1a40abc#diff-9145d7baf32c0a0158870b0f352e32e360c9cff12cd119f3aeba57dddc3d64dcL26) the attribute `placement` got removed from the `IStepOption` interfrace.

Now I get TypeScript errors in my code where I use it. I was not able to find a hint in the release notes about the removal.

The attribute is also still used here:
https://github.com/hakimio/ngx-ui-tour/blob/master/src/app/ngx-bootstrap/ngx-bootstrap.component.ts#L37

and documented here:
https://hakimio.github.io/ngx-ui-tour/ngx-bootstrap/API

Any hint what to do with this attribute? Is there an alternative to achieve the same behavior (control the placement)?

**Comment by hakimio** [comment, marked as answer]
Hi,

If you import `IStepOption` from `ngx-ui-tour-ngx-bootstrap` like its done in the demo project you linked to, the `placement` config will still be there. It was removed from the core library but it's still available in the [UI libraries](https://github.com/hakimio/ngx-ui-tour/blob/ca74d144dce5594bfc46522bbefc64ee3a95292d/projects/ngx-ui-tour-ngx-bootstrap/src/lib/step-option.interface.ts#L5).

**Reply by JoramM** [reply]
Ah, good hint. Thanks a lot for the quick answer!
